tests/format/markdown/emphasis/__snapshots__/format.test.js.snap

```diff
@@ -112,7 +112,7 @@ proseWrap: "always"
 
 1***2***3
 
-1 **_2_** 3
+1 _**2**_ 3
```

Comment:
According to [CommonMark spec](https://spec.commonmark.org/0.31.2/#emphasis-and-strong-emphasis), strong in emphasis is preferred. It get correct.
> An interpretation `<em><strong>...</strong></em>` is always preferred to `<strong><em>...</em></strong>`.